x/hard/client/cli/tx.go

```diff
@@ -119,12 +119,12 @@ func getCmdBorrow(cdc *codec.Codec) *cobra.Command {
 
 func getCmdRepay(cdc *codec.Codec) *cobra.Command {
 	return &cobra.Command{
-		Use:   "repay [1000000000ukava]",
+		Use:   "repay [amount] [owner]",
```

Comment:
Might be nicer to have owner be an optional flag, so the default syntax is 

```
kvcli tx hard repay 100000000ukava --from <key>
```

And to pay a different address' loan

```
kvcli tx hard repay 100000000ukava --owner kava1hgcfsuwc889wtdmt8pjy7qffua9dd2tralu64j --from <key>
```

In the default case, `from` becomes `owner`.

Reply:
Refactored to optional `owner` flag in https://github.com/Kava-Labs/kava/pull/801/commits/a5b1ab86e6250b010c6abb256649b549e041fe6a